app/assets/javascripts/components/users/users_page.jsx

```diff
@@ -35,10 +30,33 @@ class UsersPage extends Component {
     };
   }
 
+  // --------------------------------------------------
+  // Lifecycle
+  // --------------------------------------------------
+
+  componentWillMount() {
+    this.setState(UsersStore.getState());
+  }
+
+  componentDidMount() {
+    this._listener = UsersStore.listen((state) => this.setState(state));
+    UsersActions.fetchUsers();
+  }
+
+  componentWillUnmount() {
+    UsersStore.unlisten(this._listener);
+  }
+
+  // --------------------------------------------------
+  // Helpers
+  // --------------------------------------------------
   toggleSidebar(event) {
-    this.setState({ sidebar: !this.state.sidebar });
+    UsersActions.toggleSidebar(!this.state.sidebar);
```

Comment:
We don't need to pass `UsersActions` a sidebar state since the store already knows the state. As well, let's just remove this unnecessary `toggleSidebar` method and move the `UsersActions.toggleSidebar();` call into the arrow syntax place in the jsx below.